packages/edit-site/src/components/global-styles-sidebar/index.js

```diff
@@ -86,21 +86,17 @@ export default function GlobalStylesSidebar() {
 	}, [ shouldClearCanvasContainerView ] );
 
 	const { setIsListViewOpened } = useDispatch( editorStore );
-	const { goTo } = useNavigator();
```

Comment:
Great catch 👏 

FWIW we should also remove the `useNavigator` import.

Reply:
I just did this while testing. 👍🏻